projects/tuist/fixtures/macos_app_with_app_inside/Project.swift

```diff
@@ -22,5 +26,12 @@ let project = Project(
             infoPlist: "InnerApp/Info.plist",
             sources: ["InnerApp/Sources/**"]
         ),
+        Target(
+            name: "InnerCLI",
+            platform: .macOS,
+            product: .commandLineTool,
+            bundleId: "io.tuist.InnerCLI",
+            sources: ["InnerCLI/Sources/**"]
```

Comment:
Looks like the directory structure and glob patterns are slightly misaligned leading to a generation failure - needs a small update

```
$ tuist generate
Resolved cache profile 'Development' from Tuist's defaults
The target InnerCLI has the following invalid source files globs:
- The directory "/path/to/tuist/projects/tuist/fixtures/macos_app_with_app_inside/InnerCLI/Sources" defined in the glob pattern "/path/to/tuist/projects/tuist/fixtures/macos_app_with_app_inside/InnerCLI/Sources/**" does not exist.
Consider creating an issue using the following link: https://github.com/tuist/tuist/issues/new/choose
```